starter-code/css/main.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 10px;
+  font-family: 'Lato', sans-serif;
+  text-align: left;
+}
+
+h1 {
+  text-align: left;
+  color: #F9CA72;
+  font-size: 45px;
+  font-weight: 100;
+  margin-left: 0px;
+}
+
+h2 {
+  font-size: 30px;
+  font-weight: lighter;
+  letter-spacing: -1px;
+}
+
+h3 {
+  color: #B2B2B2;
+  font-weight: lighter;
+  font-size: 34px;
+}
+
+h4 {
+  text-align: right;
+  font-weight: bold;
+}
+
+a {
+  color: #F9CA72;
+  text-decoration: none;
+}
+
+p {
+  line-height: 150%;
+  width: 900px;
```

Comment:
Try using a % here when setting width so you don't break your site on devices that are smaller than 900px.